pod/perlclass.pod

```diff
@@ -185,6 +185,26 @@ Just like regular subroutines, methods I<can> be anonymous:
         }
     }
 
+Methods can also be declared as lexical subroutines, using the C<my> prefix.
+This creates a subroutine that is lexically visible within the current scope,
+but does not appear in the symbol table.  This creates the effect of a
```

Comment:
Nitpick: You have two sentences in a row which start with "This creates...." I suggest rephrasing one of them.

P.S. This is a great addition. Thanks, @leonerd.

Reply:
Ahyes; given it a slight edit.